packages/apps/api/test/routes/users_user_id_resend_verification_email.test.ts

```diff
@@ -0,0 +1,26 @@
+import { mockedApi, withCallerId } from './fixtures/mockedApi';
+import { randomAlphanumeric, randomExistingUser } from './fixtures/generators';
+
+describe('/users/{userId}/validate', () => {
+  describe('POST /users/{userId}/resend_verification', () => {
+    it('resends a verification email for an existing user', async () => {
+      const testUser = randomExistingUser({ password: randomAlphanumeric(10) });
+      const { api } = mockedApi([testUser]);
+      const response = await withCallerId(testUser.userId)(
+        api.post(`/users/${testUser.userId}/resend_verification`)
+      );
+
+      expect(response.statusCode).toBe(204);
```

Comment:
See comment on mock class - you're testing the endpoint behaviour here, but not yet whether it's doing what it says it does. You can do something along the lines of...

```typescript
const { api, clients } = mockedApi([testUser]);
const response = await withCallerId(testUser.userId)(
  api.post(`/users/${testUser.userId}/resend_verification`)
);

expect(response.statusCode).toBe(204);
expect(clients.auth0.resendVerificationEmail).toHaveBeenLastCalledWith(testUser.userId);
```

Reply:
This is much nicer, added.